src/shared/components/Tabs/Tabs.styles.tsx

```diff
@@ -33,6 +33,7 @@ export const TabsGroup = styled.div`
 `
 
 export const Tab = styled.div<TabProps>`
+  transition: all 0.125s ease;
```

Comment:
I think better practice is not to use `all` here, let's just add separate transitions to: `color` and `border-bottom`.